cache/refresher.go

```diff
@@ -66,15 +73,48 @@ func handleSegmentMessage(_ context.Context, msg domain.SSEMessage) error {
 	return nil
 }
 
-func handleProxyMessage(_ context.Context, msg domain.SSEMessage) error {
+func (s Refresher) handleProxyMessage(ctx context.Context, msg domain.SSEMessage) error {
 	switch msg.Event {
 	case domain.EventProxyKeyDeleted:
+		// todo
+		return nil
 	case domain.EventEnvironmentAdded:
+		if err := s.handleAddEnvironmentEvent(ctx, msg.Environments); err != nil {
+			s.log.Error(err.Error())
```

Comment:
Minor thing that could be added in the next PR but we could maybe add a more context to this log. The logger in the Proxy takes a message and then a list of key values so we could do

```go
s.log.Error("failed to handle addEnvironmentEvent", "err", err)
```